backend/backend/urls.py

```diff
@@ -117,10 +117,17 @@
 # Clone urls
 try:
 
-    import pluggable_apps.clone.urls  # noqa # pylint: disable=unused-import
```

Comment:
@harini-venkataraman why was this removed? Will it cause any issues while resolving the URL patterns?